src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -37,24 +29,32 @@ const NOCTURNAL_SECT_STATUSES = [
 	STATUSES.DIURNAL_BALANCE.id,
 ]
 
-const ACTION_STATUS_MAP = {
-	[ACTIONS.ASPECTED_BENEFIC.id]: [
-		STATUSES.ASPECTED_BENEFIC.id,
-		STATUSES.NOCTURNAL_FIELD.id,
-	],
-	[ACTIONS.ASPECTED_HELIOS.id]: [
+const SECT_BUFF_STATUSES = [...NOCTURNAL_SECT_STATUSES, ...DIURNAL_SECT_STATUSES]
+
+const ACTION_STATUS_MAP: Map<number, number[]> = new Map([
```

Comment:
This PR says "Fix Aspected Benefic's ability id", but this PR seems to do a lot more (like adding type annotations, swapping objects with a Map, etc).

I'd strongly prefer if you either changed this PR to be about type changing / cleanup and remove the ID changes (and rename the PR), or remove these type/code cleanups and PR them separately.

Reply:
Sure, I'll split them up tomorrow